code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -18,6 +18,9 @@
 	/// Ignores update_weight() if TRUE
 	var/large = FALSE
 
+	/// Belt goes over all suit slots if TRUE
+	var/special = FALSE
```

Comment:
this should be named better, `special` gives no indication as to what the var actually does. Something like `layer_over_suit` would be a better name

Reply:
Done.